### ISSUE TITLE
On clicking the EditText in the events Fragment a new activity should open

### ISSUE THREAD

**Issue by nikit19** [fossasia/open-event-android#169]
Searching for the name of the location should happen here. 
This can be re used in the Search fragment
working on it

**Comment by dreadpool2** [comment]
@nikit19 Wouldn't it be better to view it in a Fragment?

**Comment by nikit19** [comment]
I don't think so. We need a Searchable activity for this. Even eventbrite is using an activity
